meeting-notes/20181113.md

```diff
@@ -0,0 +1,203 @@
+    
+- In progress:
+  - Discussify Brand & Web Guidelines (manual)
+  - Look at new features for PeerPad (sync with Victor)
+
+ 
+#### @VictorBjelkholm
+ - Concluded:
+ 		- Error reporting on peerpad.net
+ - In Progress:
+ 		- UX review
+    - Pinner review
+ - Blocked:
+ - Next:
+    - Pinner deployment
+    - Metrics
+ 
+#### @aschmahmann
+ - Concluded:
+  - Roadmapping round #1
+  - Basic graph sync + updating working
+ - In progress:
+  - Looking into using existing CRDT tools like y.js to talk to the local Go graph syncing client
+     - Any advice would be appreciated
+ - Blocked:
+  - Not sure how much work to put into improving graph syncing vs wait for IPLD selectors/GraphSync
+  - A little blocked by the lack of the Wallet and Friends components of the Sharing API
+ - Next:
+  - More progress towards a causal chat application
+   - Which avenue to explore next somewhat depends on our file sharing API priorities
+    - Could also focus on writing proposals for this
+
+#### @jimpick
+ - Concluded:
+   - working group 2019 roadmap, 1st iteration (participant)
+   - fixes to ORMap CRDT
+   - Pedro's editor binding fixes
+   - real-life testing of peer-pad-nano with PL staff
+   - zombie gossip messages and hacks to filter them out
+   - React.js binding fixes (didn't work on Android!)
+   - Trying out the peer-pad e2e load tests
+   - Wildcard DNS / more PL staff testing
+ - In progress:
+   - Moving changes to production version (too diverged right now!)
+   - Pinner for production version?
+   - Product management for production version
+   - Review pinner work by Pedro
+   - Review pinner work by Dirk
+   - Characterize indexeddb perf issues
+   - Characterize zombie gossip issue
+   - Fix e2e smoke test
+   - Tokyo Nodefest next week!
+ - Blocked:
+   - Meetings - Roadmapping vs. PeerPad PM
+   - Dogfooding / testing
+ - Next:
+ 
+
+#### @daviddahl
+ - Concluded:
+  - see: https://github.com/IBM/ipfs-social-proof
+  - Version 0.0.2 of ipfs-social-proof
+  - Generates an ad-hoc IPFS-based "identity" based on main peer keypair
+  - User can create proofs (signed ad-hoc JSON) and post to github gist or reddit post
+  - `Autonomica` client (included) fetches the advertised proof Urls and verifies them with peer public key
+ - In progress:
+  - More tests
+  - crypto "primitives" API to support all of the requirements of IPFS Identity Manager application
+  - ASCII armoring of signatures
+  - ASCII armoring of public keys that actually work
+ - Blocked: Not really. reading all of the IPFS Identity Manager docs / requirements and W3C VC WG specs, etc. 
+ - Next:
+  - Want to support proofs and DID methods / DID formats as spec'd
+  - Extremely interested in collaboration with IPFS teams
+  
+#### Victor Grishchenko @gritzko
+
+https://github.com/gritzko/ron Replicated Object Notation (2.0,the old version) - CRDT friendly data format
```

Comment:
@mikeal, @Stebalien, @vmx and @warpfork, you will find this work interesting for the Replication front of IPLD